Add `Tournaments/Mappool` article (#8121)
ppy/osu-wiki: wiki/Tournaments/Mappool/en.md

---
stub: true
tags:
  - map pool
  - modpool
  - mod pool
  - mappool slot
  - map pool slot
  - pool slot
  - rice
  - LN
  - hybrid
  - SV
  - no mod
  - NM
  - free mod
  - force mod
  - forced mod
  - freemod
  - forcemod
  - forcedmod
  - FM
  - mixed mod
  - MM
  - tiebreaker
---

> # Mappool

Reviewer: while I'm a total outsider to the tournament scene, isn't a mappool a pool of maps, akin to the news post being a post of, well, news, and should be spelled similarly?

> A **mappool** is a collection of [beatmaps](/wiki/Beatmap) used in [tournament](/wiki/Tournaments) matches. These mappols may be divided into sections based on which [mods](/wiki/Gameplay/Game_modifier) are used, in which case they are sometimes called *modpools*, or by type of beatmap. The individual beatmaps are said to occupy a numbered *mappool slot* within each section of the mappool. People who create mappools are called *mappoolers* or *mappool selectors*, and the process itself is called *mappooling*.

Reviewer 2: ```suggestion
A **mappool** is a collection of [beatmaps](/wiki/Beatmap) used in [tournament](/wiki/Tournaments) matches. These mappools may be divided into sections based on which [mods](/wiki/Gameplay/Game_modifier) are used, in which case they are sometimes called *modpools*, or by type of beatmap. The individual beatmaps are said to occupy a numbered *mappool slot* within each section of the mappool. People who create mappools are called *mappoolers* or *mappool selectors*, and the process itself is called *mappooling*.
```

## Mappool sections

Tournaments except osu!mania ones typically include *Hidden*, *Hard Rock*, and *Double Time* sections of the mappool, which as their names suggest require the respective mods to be active. osu!mania tournaments instead use *Rice*, *LN*, *Hybrid*, and *SV*.

### No Mod

**No Mod** (***NM***) refers to not using any mods. Many tournaments require some mods to be used by default as part of their rules or format, such as [No Fail](/wiki/Game_modifier/No_Fail) or [ScoreV2](/wiki/Game_modifier/ScoreV2), which become exceptions to this notion.

### Free Mod

**Free Mod** (***FM***) refers to being free to choose any mod or mod combination. Some tournaments also provide rules that specify additional criteria such as what mods are allowed and in what combinations, or whether having no mods is allowed when Free Mod is specified.

### Force Mod

**Force Mod** (***FM***) is otherwise the same as [Free Mod](#free-mod), except that it specifies that using at least one mod is mandatory.

### Mixed Mod

> **Mixed Mod** (***MM***) means each player in the team has to pick one unique mod or mod combination each. The specific requirements may vary between tournaments.

Reviewer: I get that this is a) a stub in b) a draft PR, but is there anything else to mention about these categories at all? without any extra information, these one-liners look really slim and not deserving their special headings. I also understand that headings have their own benefits, but they don't feel warranted here

> ### Rice

Reviewer 2: It's better to unify these mania-specify tournament mappool sections. like:

- **Rice** (***RC***) is...
- **Long Note** (***LN***) is...
- **Hybrid** (***HB***) is...
- **Slider Velocity** (***SV***) is...

**Rice** is an osu!mania community term for regular notes, in reference to regular notes looking like rice. Rice maps thus contain a lot of regular notes.

### LN

**LN** in osu!mania stands for long note, and thus LN maps contain a lot of them.

### Hybrid

**Hybrid** beatmaps contain both regular and long notes, i.e. is a category in between Rice and LN.

### SV

> **SV** stands for slider velocity, which in osu!mania affects scroll speed. SV beatmaps make use of scroll speed variation gimmicks.

Reviewer: these would be all good entries for glossary (if we didn't tear it down and agree on its non-existence)...
however! we don't have articles on o!m hit objects, so these all seem like good topics to explore.

come to think of it, if your task is to describe which map pool categories are there at all, I'd try dividing them into three categories (three article sections, even)
- requiring a single specific mod
- requiring special mod selection rules (FM/NM/MM/w/e)
- containing specific map types -- here goes all o!m stuff and maybe tiebreakers (however that would be a stretch)

### Tiebreaker

**Tiebreakers** are beatmaps used to break ties. These typically focus on a variety of skills, consistent performance, and are usually longer than other such that the better player overall wins. The mod requirements vary between tournaments.

## Mappool slots

Tournament mappools are usually structured such that mappool slots correspond to specific types of beatmaps that require different skills to perform well on them. This encourages strategy and coordination when players and teams pick beatmaps in matches.

> In the [osu! game mode](/wiki/Game_mode/osu!), the the beatmap types for each slot number are standardised:

Reviewer: an
```suggestion
In the [osu! game mode](/wiki/Game_mode/osu!), the beatmap types for each slot number are standardised:
```

Reviewer 3: eh i wouldn't call this "standardized" orz

while it's true that \*most\* tournament mappools do follow this format, to my knowledge there has been any convention formalising that "this slot should be exclusively reserved for this sort of map" and so on

Reviewer 3: imo a better sentence for this would be

```suggestion
In the [osu! game mode](/wiki/Game_mode/osu!), the beatmap types for each slot number typically follow the subsequent skillset arrangement:
```

Author: yes that was the intention and is probably good to clarify
